v22.1/backup-and-restore-overview.md

```diff
@@ -20,20 +20,20 @@ For an explanation of how a backup works, see [Backup Architecture](backup-archi
 
 {% include cockroachcloud/ccloud/backup-types.md %} 
 
-## Backup and restore types
+## Backup and restore product support
 
-The following table outlines the available backup and restore types in CockroachDB. See each of the pages linked in the table for usage examples:
+This table outlines the level of product support for backup and restore features in CockroachDB. See each of the pages linked in the table for usage examples:
```

Comment:
I found the "level of product support" to be a bit unclear... How about something like "This table outlines the backup & restore features that are supported by the various CockroachDB products."